README.md

```diff
@@ -257,6 +257,34 @@ If you enable ProGuard, then add this rules in your ProGuard file.
 ```
 
 
+## Support TLS 1.2 for older Android 5.0
+
+To follow PCI compliance will support TLS 1.2 or newer version on our vault. The applications are using Omise Android SDK and running older Android 5.0 will stop working with regardless. Because Android API older version 20 disabled TLS 1.2 by default for openssl security reason. To keep Omise Android SDK working on older Android 5.0 developers need to use Google Play Services to install a new patch of SSL Security Provider.
+ 
+If use already use one of Google Play Services e.g. **GCM**, **Analytics** you already to use `ProviderInstaller` from them. But if not developers need to add one of Google Play Services to project. 
+
+```gradle
+compile 'com.google.android.gms:play-services-analytics:16.0.3'
+```
+
+After already add Google Play Services to the project. Developers need to install Google Play Services to the Application class or in the Activity before start calling Omise Android SDK. Developers can install by follow this snippet code.
+
+```java
+
+if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.KITKAT) {
+    try {
+        ProviderInstaller.installIfNeeded(this);
+    } catch (GooglePlayServicesRepairableException e) {
+        e.printStackTrace();
+    } catch (GooglePlayServicesNotAvailableException e) {
+        e.printStackTrace();
+    }
+}
+```
+
+> **Note:** This will support only the devices that have Google Play Services installed on the device.
+
+
```

Comment:
There are a few things that can be improved here so it's just easier for me to copy the whole thing here and add the corrections 👍 

## Support TLS 1.2 for devices with OS older than Android 5.0  

To follow PCI compliance we will support TLS 1.2 or newer on our vault. The applications that are using Omise Android SDK and are running on OS versions older than Android 5.0 will stop working. This is because Android API older than version 20 have TLS 1.2 disabled by default for openssl security reason. To keep Omise Android SDK working on Android devices with versions older than 5.0, developers need to use Google Play Services to install a new patch of SSL Security Provider.
 
If use already use one of Google Play Services libraries e.g. **GCM**, **Analytics** you are already able to use `ProviderInstaller` from them. But if not developers need to add one of the Google Play Services libraries to their project. 
 ```gradle
compile 'com.google.android.gms:play-services-analytics:16.0.3'
```
 After adding Google Play Services to the project, developers need to install Google Play Services in the Application class or in the Activity before they start to call Omise Android SDK. Developers can install that by following this snippet of code.
 ```java
 if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.KITKAT) {
    try {
        ProviderInstaller.installIfNeeded(this);
    } catch (GooglePlayServicesRepairableException e) {
        e.printStackTrace();
    } catch (GooglePlayServicesNotAvailableException e) {
        e.printStackTrace();
    }
}
```
 > **Note:** This will only support the devices that have Google Play Services installed on them.

Reply:
@Namiii Thank you. I will wait for other feedback then will combine them.

Reply:
Here's my version of the note

## Note on TLS 1.2
PCI-DSS standard requires the service to communicate in TLS 1.2 or higher using strong encryption suite. This means that every clients must connect to Omise service with TLS 1.2 or higher using some valid strong encryption suites. However TLS 1.2 in Android is vary depends on the Android OS

#### Android API 21 or higher
You can use our SDK without any change. The SDK already fully supports communication to the Omise Service using TLS 1.2 with the properly encryption suite.
#### Android API 16 to API 20 with Google Play Services
These Android API versions don't support the proper encryption suites out of the box. However `Google Play Service` has at the `ProviderInstaller` API to add support for the proper encryption suite. You may already use the Google Play Service in your app already; Google Play Service includes many common libraries used in many Android apps including GCM, Analytics and more. You can add support for the proper encryption suite with Google Play Service by doing the following steps
1. Add Google Play Service Analytics to your app grade setting
```gradle
 compile 'com.google.android.gms:play-services-analytics:16.0.3'
 ```
2. Asks the ProviderInstaller to install the encryption suites with the follow code
```java
 if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.KITKAT) {
     try {
         ProviderInstaller.installIfNeeded(this);
     } catch (GooglePlayServicesRepairableException e) {
         e.printStackTrace();
     } catch (GooglePlayServicesNotAvailableException e) {
         e.printStackTrace();
     }
 }
 ```
> **Note:** Google Play Service may not available on every brands or models. Please be concerned about the Google Play Service compatibility
#### API 15 or lower
These Android API levels could not add support for the proper encryption suites easily which means that the Android devices running those OS versions may not be able to use to connect to many services on the internet. We recommend you to drop support for those API levels